vaccine-standby/functions/setup.js

```diff
@@ -0,0 +1,507 @@
+/* eslint-disable no-console, func-names */
+exports.handler = async function (context, event, callback) {
+  const flowDefinition = {
```

Comment:
Open question: do you think it would make sense to move this into a `json` private asset instead and load it in here? That way it might be easier for people to change the JSON. But I'm not fully sure if it's needed

Reply:
+1

I find coming to this file overwhelming, even though there's only 50 lines of code with 450 lines of configuration. I assume that this JavaScript object is exported from an example Studio Flow, so I think it would be more maintainable if it lived in its own file, which can be overwritten if you need to update the flow, and would get out of the way of the code in this function.

Reply:
Agreed @dkundel @philnash but how do you reference a private asset file from inside a function?

Reply:
@dkundel @philnash this is now addressed